index.js

```diff
@@ -56,6 +56,7 @@ module.exports = {
                                       */
     "key-spacing": "off",
     "linebreak-style": "off",
+    "lines-between-class-members": ["error", "always", { "exceptAfterSingleLine": true }],
```

Comment:
I assume this is making the rule more lenient?

Reply:
Yes, it's more lenient. To translate to a complete sentence: "When there's a class member that's just a single line, I don't need a blank line between it and the next class member."

Reply:
Oh, that's nice!